src/API/NoteActions.php

```diff
@@ -115,6 +115,24 @@ public function trigger_note_action( $request ) {
 			$tracks_event = 'wcadmin_inbox_action_click';
 		}
 
+		$screen_name = '';
```

Comment:
Optional, but we can extract this block to its own function which might make it easier to follow or reuse going forward.  E.g., `get_screen_name()`.

Reply:
Good idea! Sounds good to me, I modified this in the commit e5fdd11